### PR TITLE
qa/cephfs: kill mount if it gets evicted by mds

### PR DESCRIPTION
**Pull request by ukernel**
otherwise, teardown() hange at umount

### CODE REVIEW

File: qa/tasks/cephfs/test_client_recovery.py [hunk 1]
```diff
@@ -395,14 +395,21 @@ def test_filelock_eviction(self):
...
+            mount_a_client_id = self.mount_a.get_global_id()
+            self.fs.mds_asok(['session', 'evict', "%s" % mount_a_client_id])
 
-        # Evicting mount_a should let mount_b's attepmt to take the lock
-        # suceed
-        self.wait_until_true(
-            lambda: lock_taker.finished,
-            timeout=10)
+            # Evicting mount_a should let mount_b's attepmt to take the lock
+            # suceed
```
Comment (gmayyyha) [inline qa/tasks/cephfs/test_client_recovery.py]: L403: comment typo 'succeed' ?

Comment (amitkumar50) [reply qa/tasks/cephfs/test_client_recovery.py]: NIT: wrong spelling of 'attepmt'.

Even why this comment is highlighted as changed?

**Comment by ukernel** [comment]
http://pulpito.ceph.com/zyan-2017-09-13_09:05:41-kcephfs-master-testing-basic-mira/

tests failed with FS_DEGRADED in cluster log. can be considered as passed;)

**Comment by batrick** [comment]
http://tracker.ceph.com/issues/21275

**Pull request merged**